app/navigation/Navigator.scala

```diff
@@ -75,7 +75,8 @@ class Navigator @Inject()(
     //  Trustees
     case IsThisLeadTrusteePage(index) => _ =>_ => controllers.register.trustees.routes.TrusteeIndividualOrBusinessController.onPageLoad(NormalMode, index, draftId)
     case TrusteeIndividualOrBusinessPage(index)  => _ => ua => trusteeIndividualOrBusinessRoute(ua, index, draftId)
-    case TrusteeUtrYesNoPage(index) => _ => ua => trusteeUtrYesNoRoute(ua, index, draftId)
+    case TrusteeUtrYesNoPage(index) => _ => _ => controllers.register.trustees.routes.TrusteeBusinessNameController.onPageLoad(NormalMode, index, draftId)
+    case TrusteeOrgNamePage(index) => _ => _ => controllers.register.trustees.routes.TrusteeBusinessNameController.onPageLoad(NormalMode, index, draftId)
 
```

Comment:
routing to self.  Because of unfinished story?

Reply:
@JoeSimmonds correct, will update route for this page once the next page in the journey has been created.

Reply:
I've also committed an update to correct the required answer redirect for the 'Is this trustee a UK registered company' page.